spec.ts

```diff
@@ -0,0 +1,83 @@
+import { browser, element, by } from 'protractor';
+
+import { YM } from "./epochconverter/convert";
+import { SEC } from './epochconverter/convert_sdhm';
+import { DTG } from './epochconverter/timestamp';
+import { YMDHMS } from './epochconverter/ymdhms_timestamp';
+import * as moment from 'moment';
+
+
+describe('Epochconverter with protractor -', function() {
+
+	// ########## Mein ToDo ##########
+	// GMT zu Local time wechseln
+	// Timestamp to Human date fehlt noch
+	// Jeweils Rückkgabe nach Buttonklick auslesen
+	// ###############################
+
+	// Date and time
+	let year = moment().format('Y');
+	let month = moment().format('M');
+	let day = moment().format('D');
+	let hours_utc = moment().utc().format('H');
+	let hours = moment().format('H');
+	let minutes = moment().format('m');
+	let seconds = moment().format('s');
+
+	// letiables for test 
+	let ymdhms_timestamp_year = element(by.id("hf-d1")).element(by.className("dateform year"));
```

Comment:
keine Locators mehr in den tests, diese sollten in den Page Objects definiert werden

Reply:
Diese Variablen braucht man für expect toEqual, ist alle notige Variablen, den Rest habe ich schon entfernt.

Reply:
genau diese Sachen sollen aber in die Page Objects ausgelagert werden. Aktuell pflegst du diese Werte 2 Mal und hast dadurch deinen Wartungsaufwand erhöht, anstatt verringert. Unbedingt umbauen!